cg/src/app.js

```diff
@@ -8,6 +8,11 @@ const expressWs = require('express-ws');
 const { apiErrorHandler } = require('./utils/errors');
 const { errors } = require('celebrate');
 
+
+const corsOptions = {
```

Comment:
@Jakedavies I seem to have missed this wasn't passed to `cors()` on line 15. Ideally we should also have a test running `OPTIONS` requests